test-integration/helpers/describe.coffee

```diff
@@ -49,12 +51,31 @@ describe = (name, cb) ->
       # Wait 20sec for the browser to start up
       @timeout(20 * 1000, true)
 
+      chromeOptions = new chrome.Options()
+      prefs = {
+        download:
+          "default_directory": "test-integration/downloaded-files"
+      }
+      chromeOptions.setUserPreferences(prefs)
+
+      firefoxProfile = new firefox.Profile()
+      firefoxProfile.setPreference("browser.helperApps.neverAsk.saveToDisk", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+      firefoxProfile.setPreference("browser.helperApps.neverAsk.openFile", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+      firefoxProfile.setPreference("browser.helperApps.alwaysAsk.force", false)
+      firefoxProfile.setPreference("browser.download.useDownloadDir", true)
+      firefoxProfile.setPreference("browser.download.manager.showWhenStarting",false)
+      firefoxProfile.setPreference("browser.download.folderList", 2)
+      firefoxProfile.setPreference("browser.download.dir", "#{__dirname}/../downloaded-files")
```

Comment:
Could we pull out `"#{__dirname}/../downloaded-files"` and set it as a constant on the `@` -- `@dowloadDirectory = "#{__dirname}/../downloaded-files"` for example.  This way, we could use the same variable on [line 57](https://github.com/openstax/tutor-js/pull/996/files#diff-d44a9804e9150c63459506961fbeda1fR57) as well, and even access it as `@test.driver.dowloadDirectory` on [line 61 in `test-integration/helpers/scores.coffee`](https://github.com/openstax/tutor-js/pull/996/files#diff-9aa463384df46c7ae7cf08eba2dd8c9bR61) where you're checking if the file exists